packages/search-ui-site-search-connector/src/request.ts

```diff
@@ -26,7 +31,10 @@ export default async function request(engineKey, method, path, params) {
   if (response.status >= 200 && response.status < 300) {
     return json;
   } else {
-    const message = json && json.error ? json.error : response.status;
-    throw new Error(message);
+    const message =
+      json && json.errors && Object.entries(json.errors).length > 0
+        ? JSON.stringify(json.errors)
+        : response.status;
+    throw new Error(`${message}`);
```

Comment:
Had to change the code slightly here, since based on docs API returns "errors" field, not "error" field.